Sources/MapboxNavigation/FollowingCameraOptions.swift

```diff
@@ -0,0 +1,118 @@
+     
+     If `false`, `NavigationViewportDataSource` will not modify `CameraOptions.center` property.
+     
+     Defaults to `true`.
+     */
+    public var centerUpdatesAllowed = true
+    
+    /**
+     If `true`, `NavigationViewportDataSource` will continuously modify `CameraOptions.zoom` property
+     when producing camera frame in `NavigationCameraState.following` state.
+     
+     If `false`, `NavigationViewportDataSource` will not modify `CameraOptions.zoom` property.
+     
+     Defaults to `true`.
+     */
+    public var zoomUpdatesAllowed = true
+    
+    /**
+     If `true`, `NavigationViewportDataSource` will continuously modify `CameraOptions.bearing` property
+     when producing camera frame in `NavigationCameraState.following` state.
+     
+     If `false`, `NavigationViewportDataSource` will not modify `CameraOptions.bearing` property.
+     
+     Defaults to `true`.
+     */
+    public var bearingUpdatesAllowed = true
+    
+    /**
+     If `true`, `NavigationViewportDataSource` will continuously modify `CameraOptions.pitch` property
+     when producing camera frame in `NavigationCameraState.following` state.
+     
+     If `false`, `NavigationViewportDataSource` will not modify `CameraOptions.pitch` property.
+     
+     Defaults to `true`.
+     */
+    public var pitchUpdatesAllowed = true
+    
+    /**
+     If `true`, `NavigationViewportDataSource` will continuously modify `CameraOptions.padding` property
+     when producing camera frame in `NavigationCameraState.following` state.
+     
+     If `false`, `NavigationViewportDataSource` will not modify `CameraOptions.padding` property.
+     
+     Defaults to `true`.
+     */
+    public var paddingUpdatesAllowed = true
+    
+    /**
+     Options, which allow to modify `CameraOptions.bearing` property based on information about
+     bearing of an upcoming maneuver.
+     */
+    public var bearingSmoothing: BearingSmoothing = BearingSmoothing()
+    
+    /**
+     Options, which allow to modify framed route geometries by appending additional coordinates after
+     maneuver to extend the view.
+     */
+    public var frameGeometryAfterManeuver: FrameGeometryAfterManeuver = FrameGeometryAfterManeuver()
+}
+
+/**
+ Options, which allow to modify `CameraOptions.bearing` property based on information about
+ bearing of an upcoming maneuver.
+ */
+public struct BearingSmoothing {
+    
+    /**
+     Controls how much the bearing can deviate from the location's bearing, in degrees.
+     
+     In case if set, the `bearing` property of `CameraOptions` during active guidance navigation
+     won't exactly reflect the bearing returned by the location, but will also be affected by the
+     direction to the upcoming framed geometry, to maximize the viewable area.
+     
+     Defaults to `45.0` degrees.
+     */
+    public var maximumBearingSmoothingAngle: CLLocationDirection? = 45.0
```

Comment:
20° was too small?

Reply:
Was changed for parity with Android, it's set to `45.0` degrees there.